File: CursoPOOUber/Csharp/Payment.cs
using System;

namespace PaymentTypes
{

    //Definition of the "Payment" class.
    public class Payment
    {

        //Attributes
        public int Id { get; set; }

        public Payment(int id)
        {
            this.Id = id;
        }

        public void PrintPaymentData()
        {
            Console.WriteLine("This is the ID of this payment: " + Id);
        }
    }
}

File: CursoPOOUber/Java/Payment.java
package CursoPOOUber.Java;

//Definition of the "Payment" class.
public class Payment {
    Integer id;

    //Attributes
    public Payment(Integer id) {
        this.id = id;
    }

    void printPaymentData() {
        System.out.println("This is the ID of this payment: " + id);
    }
}
